QgisModelBaker/utils/gui_utils.py

```diff
@@ -490,9 +491,10 @@ def refresh_model(self, source_model, db_connector=None, silent=False):
 
     def _transfer_file_models(self, xtf_file_path):
```

Comment:
```suggestion
    def _transfer_file_models(self, transfer_file_path):
```
The parameter should be renamed if it can be both an xtf or an itf file